src/lib/strava/api.ts

```diff
@@ -46,25 +57,22 @@ export async function authorize(params: any): Promise<Token> {
   const data = await res.json();
 
   if (!res.ok) {
-    console.log('Authorization fail', data.data);
+    console.log(`token ${type} fail`, data.data);
```

Comment:
perhaps here we should auto logout the user and trigger an Alert to relogin? probably something for a future PR though.

Reply:
Yeah not sure what we do on refresh failure. It'll raise to the caller, the caller should alert the user if necessary or try again.